packages/arb-bridge-peripherals/contracts/tokenbridge/test/TestCustomTokenL1.sol

```diff
@@ -20,7 +20,7 @@ interface IL1CustomGateway {
     ) external payable returns (uint256);
 }
 
-interface IGatewayRouter {
+interface IGatewayRouter2 {
```

Comment:
fyi doing this because a compiler bug(?)
```
contracts/tokenbridge/test/TestCustomTokenL1.sol:84:9: TypeError: Member "setGateway" not unique after argument-dependent lookup in contract IL1GatewayRouter.
        IL1GatewayRouter(router).setGateway{ value: valueForRouter }(bridge, m ...
        ^---------------------------------^
```